pkg/printers/internalversion/printers.go

```diff
@@ -1659,10 +1659,18 @@ func printEvent(obj *api.Event, options printers.GenerateOptions) ([]metav1.Tabl
 		Object: runtime.RawExtension{Object: obj},
 	}
 
-	firstTimestamp := translateTimestampSince(obj.FirstTimestamp)
-	lastTimestamp := translateTimestampSince(obj.LastTimestamp)
+	var firstTimestamp string
+	var lastTimestamp string
+	if obj.FirstTimestamp.IsZero() {
+		firstTimestamp = translateTimestampSince(obj.CreationTimestamp)
```

Comment:
use `eventTime` better. 
 In v1beta1.event group, `eventTime` replaces `firstTimestamp`